content/concepts/products.md

```diff
@@ -871,14 +871,14 @@ Note that the `family` field is only available since the 3.2 version.
 :::
 
 ::: warning
-Note that the `quality_scores` field is **only available on SaaS platforms** and when the `with_quality_scores` query parameter is set to `true`.
+Note that the `quality_scores` field is **only available since 7.0** and when the `with_quality_scores` query parameter is set to `true`.
```

Comment:
```suggestion
Note that the `quality_scores` field is **only available since the 7.0 version** and when the `with_quality_scores` query parameter is set to `true`.
```